docs/users-guide/09-multi-series-charting.md

```diff
@@ -7,73 +7,79 @@ Data in isolation is rarely all that useful. One of the best ways to add context
 - New users per day vs. returning users per day.
 - Orders per day from a few different product lines.
 
-### In Metabase there are two main ways to get data side-by-side.
+### Two ways to display data side-by-side
 
-1. Combining two existing saved questions that share a common dimension (like time) on a dashboard
+1. [**Use the notebook editor to ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) (or in SQL, if you’re fancy). Example: the count of users by region over time.
```

Comment:
You can add multiple dimensions in both Simple and Custom mode. In Simple mode, you do it by clicking the + next to a column in the Group By part of the Summarize sidebar.

```suggestion
1. [**Ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) with the Simple or Custom query builders (or in SQL, if you’re fancy). Example: the count of users by region over time.
```